### PR TITLE
feat: add reward/incentive claiming to cl script

### PR DESCRIPTION
**Pull request by czarcas7ic**
Closes: #XXX

## What is the purpose of the change

Adds both spread reward and incentive claiming to the cl script. For both rewards:
- Selects a single random account
- Retrieves all of that address's position IDs
- Selects a random subset of those position IDs
- Claims for all those position IDs

### CODE REVIEW

File: tests/cl-go-client/main.go [hunk 1]
```diff
@@ -48,13 +48,19 @@ const (
...
+	// claimIncentivesOperation claims a random subset of incentives from a random account.
+	claimIncentivesOperation
 )
 
 const (
 	expectedPoolId           uint64 = 1
 	addressPrefix                   = "osmo"
 	localosmosisFromHomePath        = "/.osmosisd-local"
-	consensusFee                    = "1500uosmo"
+	consensusFee                    = "3000uosmo"
```
Comment (czarcas7ic) [inline tests/cl-go-client/main.go:63]: note to reviewer: needed when we claim rewards for many position IDs at once

File: tests/cl-go-client/main.go [hunk 2]
```diff
@@ -561,6 +572,106 @@ func createPoolOp(igniteClient cosmosclient.Client) {
...
+	// Instantiate a query client
+	clClient := clqueryproto.NewQueryClient(igniteClient.Context())
+
+	accountMutex.Lock() // Lock access to getAccountAddressFromKeyring
+	senderAddress := getAccountAddressFromKeyring(igniteClient, accountName)
+	accountMutex.Unlock() // Unlock access to getAccountAddressFromKeyring
+
+	userPositionResp, err := clClient.UserPositions(context.Background(), &clqueryproto.UserPositionsRequest{
+		Address: senderAddress,
+		PoolId:  0,
```
Comment (stackman27) [inline tests/cl-go-client/main.go:590]: Is this default poolId that always contain positions?

**Review by stackman27 — APPROVED** [review]
LGTM, just have a small question

File: tests/cl-go-client/main.go (hunk 2 above)
Comment (czarcas7ic) [reply tests/cl-go-client/main.go:590]: this lists all positions instead of just a specific pool

**Pull request merged**